numpydoc/docscrape_sphinx.py

```diff
@@ -225,6 +225,7 @@ def __str__(self, indent=0, func_role="obj"):
         out += self._str_extended_summary()
         out += self._str_param_list('Parameters')
         out += self._str_returns()
```

Comment:
I know it is not required, but for readability it might be worth passing `'Returns'` here.  This is really minor, ignore it if you want.